### ISSUE TITLE
Unknown column 'reacted_by' in 'where clause'

### ISSUE THREAD

**Issue by salvatormundiz** [cybercog/laravel-love#254]
I working with V9 laravel-love!
my User model use as Reacterable and Reactable!
when I use this line:
```
$user=User::find(6);
        $myTherapist = User::query()
            ->whereReactedBy($user)
            ->get();
        dd($myTherapist);
```
my response is:
```
SQLSTATE[42S22]: Column not found: 1054 Unknown column 'reacted_by' in 'where clause'
```

**Comment by antonkomarev** [comment]
@salvatormundiz did you added `Reactable` trait to User model? 

https://laravel-love.readme.io/docs/setup-reactable

**Comment by salvatormundiz** [comment]
It is my User model I want user can bookmark other user so use both:
```
use Cog\Contracts\Love\Reacterable\Models\Reacterable as ReacterableInterface;
use Cog\Laravel\Love\Reacterable\Models\Traits\Reacterable;
use Cog\Contracts\Love\Reactable\Models\Reactable as ReactableInterface;
use Cog\Laravel\Love\Reactable\Models\Traits\Reactable;
class User extends Authenticatable implements ReacterableInterface,ReactableInterface
{
    use HasApiTokens, HasFactory, Notifiable,HasRoles,Reacterable,Reactable;
    ```

**Comment by antonkomarev** [comment]
@salvatormundiz I forgot to add to the documentation query scopes.

- Was implemented in #227

**Comment by antonkomarev** [comment]
@salvatormundiz I've described setup process in following pages:
- https://laravel-love.readme.io/docs/setup-reacterable
- https://laravel-love.readme.io/docs/setup-reactable

And updated usage pages:
- https://laravel-love.readme.io/docs/reacterable-filtering
- https://laravel-love.readme.io/docs/reactable-ordering
- https://laravel-love.readme.io/docs/reactable-filtering

Thanks for pointing this issue!
